src/app/services/genres/genres-holder.service.ts

```diff
@@ -0,0 +1,27 @@
+import {Injectable} from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class GenresHolderService {
+  private _genres: string[] = [
+    "All",
+    "Drama",
+    "Romance",
+    "Animation",
+    "Adventure",
+    "Family",
+    "Comedy",
+    "Fantasy",
+    "Science Fiction",
+    "Action"
+  ]
+
+  constructor() {
+  }
+
+
+  get genres(): string[] {
```

Comment:
why not just make genres a public prop? I can mutate the private _genres anyway using the GenresHolderService.genres.push(something), so it can be simplified